integreat_cms/release_notes/current/unreleased/3098.yml

```
@@ -0,0 +1,2 @@
+en: Fix empty Zammad URL in region form
```

```suggestion
en: Fix error when creating region without Zammad URL 
```
Nit: I think this is a bit better to understand for people without a background in IT?